.github/workflows/handle-new-contribution.yml

```diff
@@ -0,0 +1,53 @@
+name: Handle New Contribution
+
+on:
+  pull_request_target:
+    types: [opened, ready_for_review]
+
+permissions:
+  pull-requests: write
+  contents: write
+
+jobs:
+  assign_and_label:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.head.repo.fork == true
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+      - name: Print Context
+        run: |
+          echo "$GITHUB_CONTEXT"
+        env:
+          GITHUB_CONTEXT: ${{ toJson(github) }}
+      - name: Add an assignee and a Contribution label
+        env:
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+          PR_URL: ${{github.event.pull_request.html_url}}
+        run: |
+          echo "get content_roles.json using curl"
+          content_roles=$( curl https://raw.githubusercontent.com/demisto/content/master/.github/content_roles.json )
+          echo "$content_roles"
+          
+          echo "==== $(date): Starting pipenv setup... ===="
```

Comment:
all of this seems like a lot of work for something simple. We dont need the python here, or the files
We can use jq for this

`curl theurl | jq .CONTRIBUTION_TL` will give us the tl name